specs/metricbeat.spec.yml

```
@@ -287,6 +287,13 @@ inputs:
     shippers: *shippers
     command:
       args: *args
+  - name: statsd/metrics
+    description: "Statsd metrics"
+    platforms: *platforms
+    outputs: *outputs
+    shippers: *shippers
+    command:
+      args: *args
```

```suggestion
    command: *command
```

This is the format you want to keep when the conflict is resolved.

---

done